OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -563,6 +561,14 @@ void testSpringMass()
     PointToPointSpring *copyOfSpring = spring.clone();
 
     ASSERT(*copyOfSpring == spring);
+
+    // Verify that the PointToPointSpring is correctly deserialized from
+    // previous major version of OpenSim.
+    Model bouncer("bouncing_block_30000.osim");
+    SimTK::State& s = bouncer.initSystem();
+    bouncer.realizeAcceleration(s);
+
+    Vec3 comA = bouncer.calcMassCenterAcceleration(s);
```

Comment:
I think this line will generate a warning because `comA` is not used. In this situation, I have seen
```cpp
/*Vec3 comA =*/ bouncer.calcMassCenterAcceleration(s);
```
elsewhere in the repo.